root/static/scripts/common/components/AreaWithContainmentLink.js

```diff
@@ -13,6 +13,7 @@ import EntityLink from './EntityLink.js';
 
 component AreaWithContainmentLink(
   area: AreaT,
+  showCreditedAs: boolean = true,
```

Comment:
I don't think that areas can have credited-as text, but I added this since this component uses `EntityLink` and `DescriptiveLink` currently passes the same set of properties to this component and directly to `EntityLink`.